.travis.yml

```diff
@@ -10,18 +10,17 @@ notifications:
       - ros-contributions@amazon.com
       - travis-build@platform-notifications.robomaker.aws.a2z.com
 env:
-  matrix:
-    - ROS_DISTRO="kinetic" ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
-    - ROS_DISTRO="kinetic" ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
-    - ROS_DISTRO="melodic" ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
-    - ROS_DISTRO="melodic" ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
   global:
-    - AFTER_SCRIPT='if [ -z "$PRERELEASE" ] || [ "$PRERELEASE" != true ]; then cd ~/catkin_ws && catkin build --pre-clean --catkin-make-args run_tests && catkin_test_results --verbose; fi;'
-matrix:
-  allow_failures:
-    - env: ROS_DISTRO="melodic" ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
-    - env: ROS_DISTRO="melodic" ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
+    - GITHUB_ACCOUNT=aws-robotics
+    - PACKAGE_NAME=aws_common
+  matrix:
+    # define DOWNSTREAM_GITHUB_REPO=name-of-your-repo if you'd like to trigger downstream repos' travis builds
+    - ROS_DISTRO=kinetic ROS_VERSION=1
+    - ROS_DISTRO=lunar   ROS_VERSION=1
```

Comment:
Are we still doing lunar? Doesn't look like we're testing it for other packages.